docs/distributed/README.md

```diff
@@ -1,74 +1,85 @@
+Distributed Minio provides protection against multiple node/drive failures and [bit rot](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide.html#whatisrot) using [erasure code](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide). Since distributed Minio and erasure code both require a minimum of four hard drives, erasure code is automatically used when distributed Minio is launched.
 
-### High availability
+### High Availability
 
-A stand-alone Minio server would go down if the server hosting the disks goes offline. In contrast, a distributed Minio setup with _n_ disks will have your data safe as long as _n/2_ or more disks are online. You'll need a minimum of _(n/2 + 1)_ [Quorum](https://github.com/minio/dsync#lock-process) disks to create new objects though.
+A standalone server running Minio Server will go down if the hard drives go offline. In contrast, a distributed Minio configuration with N disks will keep data safe as long as N/2 or more hard drives are online. This requires a minimum of (N/2 + 1) [Quorum](https://github.com/minio/dsync#lock-process) hard drives to create new objects.
 
-For example, an 8-node distributed Minio setup with 1 disk per node would continue serving files, even if up to 4 disks are offline. But, you'll need at least 5 disks online to create new objects.
+For example, an 8-node distributed Minio configuration with one hard drive per node will continue serving files even if as many as 4 hard drives are offline. This configuration requires at least 5 hard drives online to create new objects.
 
 ### Limits
 
-As with Minio in stand-alone mode, distributed Minio has a per tenant limit of minimum 2 and maximum 32 servers. There are no limits on number of disks shared across these servers. If you need a multiple tenant setup, you can easily spin up multiple Minio instances managed by orchestration tools like Kubernetes.
+As with Minio running in standalone mode, distributed Minio supports 2-32 servers per tenant. There are no limits on the number of hard drives shared across these servers. For a multi-tenant configuration, orchestrate multiple Minio instances using a tool like Kubernetes.
 
-Note that with distributed Minio you can play around with the number of nodes and drives as long as the limits are adhered to. For example, you can have 2 nodes with 4 drives each, 4 nodes with 4 drives each, 8 nodes with 2 drives each, 32 servers with 24 drives each and so on.
+**Note:** With distributed Minio, the number of nodes and hard drives can be adjusted within the limits. For example, a configuration can have 2 nodes with 4 hard drives each, 4 nodes with 4 hard drives each, 8 nodes with 2 hard drives each, 32 servers with 24 hard drives each, etc.
```

Comment:
Need to replace adjusted, it implies a cluster can have nodes added and removed.